#BE-3169 Release Notes for 3.15.0 (#439)
appcircleio/appcircle-docusaurus: docs/updates/release-notes.md

- The uploaded or built AAB files are now automatically converted to a universal format. Additionally, the newly created APK file is saved discreetly with the second app's resource id. <CloudBadge/>
- A new command, ["build active-list"](https://docs.appcircle.io/appcircle-api/) has been added, allowing users to view active builds in the queue directly from their command line interface. <CloudBadge/> <SelfHostedBadge/>
- A new command, ["build view"](https://docs.appcircle.io/appcircle-api/) has been added, enabling users to access and view detailed information about builds directly from the command line interface. <CloudBadge/> <SelfHostedBadge/>
- The "Default M1 pool" now includes [Xcode 15.4 beta-1](https://docs.appcircle.io/infrastructure/ios-build-infrastructure#available-xcode-versions) installed on runners. As this is a beta release, we strongly recommend testing your workflows extensively to ensure compatibility and stability. <CloudBadge/> <SelfHostedBadge/>

### :muscle: Improvements

> - New [Publish Events](https://docs.appcircle.io/publish-module/), such as Publish Profile Created, Publish Profile Deleted, App Version Uploaded, App Version Created, and App Version Deleted, have been introduced to enrich the activity report. <CloudBadge/>

Reviewer: ```suggestion
- We have introduced new [Publish Events](https://docs.appcircle.io/publish-module/) like Publish Profile Created, Publish Profile Deleted, App Version Uploaded, App Version Created, and App Version Deleted to enrich the activity report. <CloudBadge/>
```

Reviewer: When creating documentation, please ensure you use active voice wherever possible. This style makes our documentation clearer and more engaging by directly identifying the action and who is performing it. Active voice contributes to more straightforward and effective communication, enhancing the readability and accessibility of our content for all users.

> - Branch search and filter operations have been fine-tuned for faster performance and smoother functionality. <CloudBadge/>

Reviewer: ```suggestion
- We have fine-tuned branch search and filter operations for faster performance and smoother functionality. <CloudBadge/>
```

> - The capability to store profile pins (Enterprise store profile, Distribution profile, Build Profile, and Publish Profile) for active users on an individual basis has been introduced. Previously, profile pins were stored solely on an organization-wide level. Now, each active user can set and manage their own pin independently, providing greater flexibility and customization options. <CloudBadge/>

Reviewer: ```suggestion
- We have introduced the capability for active users to store profile pins (Enterprise store profile, Distribution profile, Build Profile, and Publish Profile) individually. Previously, profile pins were stored solely on an organization-wide level. Now, each active user can set and manage their own pin independently, providing greater flexibility and customization options. <CloudBadge/>
```

- Added [Okta](https://docs.appcircle.io/account/my-organization/sso-providers-configuration/okta-saml) tile based login so users can log in to the Appcircle dashboard by clicking the Appcircle app icon on Okta side. <CloudBadge/>

### 🐞 Fixes

> - Fixed a bug where the report would only update once due to discrepancies between canceling after the build starts and canceling before it starts.<CloudBadge/>

Reviewer: ```suggestion
- We have fixed a bug that caused the report to update only once due to discrepancies between canceling after the build starts and canceling before it starts. <CloudBadge/>
```
